### PR TITLE
my bykepage

### PR DESCRIPTION
**Pull request by vinasxz**

### CODE REVIEW

File: src/style/menu.scss
Comment (EdPirro) [inline src/style/menu.scss:1]: You should be able to scroll the menu if the viewport is shorter than it [Referece](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Make%20it%20possible%20to%20scroll%20the%20menu%20if%20it%20is%20higher%20than%20the%20viewport)

File: src/index.html [hunk 1]
```diff
@@ -1,22 +1,438 @@
...
+                    name="name"
+                    placeholder="Name"
+                    required
+                  />
+
+                  <input
+                    class="form__field"
+                    type="email"
+                    name="email"
+                    placeholder="Email"
```
Comment (EdPirro) [inline src/index.html:375]: Placeholder should be an example of a valid value for the field [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Placeholders%20should%20give%20examples%20of%20what%20should%20be%20added%2C%20to%20make%20the%20expected%20format%20clear)

File: src/scripts/main.js
Comment (EdPirro) [inline src/scripts/main.js:1]: Could be implemented without js using the `:has` pseudo class [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Disable%20page%20scrolling%20under%20the%20menu%20using%20the%20next%20code):

```css
.page:has(.page__menu:target) {
  overflow: hidden; /* not to scroll the page */
}
```

File: src/index.html [hunk 2]
```diff
@@ -1,22 +1,438 @@
...
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#contacts"
+                >
+                  Contacts
+                </a>
+              </li>
```
Comment (EdPirro) [inline src/index.html:135]: Add an empty line after multiline sibling elements [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Add%20empty%20lines%20between%20multiline%20sibling%20blocks%20of%20HTML)

**Review by EdPirro — CHANGES_REQUESTED** [review]
Awesome job, @vinasxz! Great attention to details and clean code! 👏. I spotted just a few improvements and please, remember to add a link to your deployment in the PR description [Reference](https://mate-academy.github.io/layout_task-guideline/#create-a-pull-request-pr)

File: src/index.html (hunk 2 above)
Comment (vinasxz) [reply src/index.html]: Prettier configs, if you have any with "good practice" better then i'm already using, please send to me.

**Review by jv-aquino — APPROVED** [review]
Wow Vinicius, awesome job! Your hover transitions are clean, the file structure is perfect! Really no comments, just an awesome job! It will look great in your portfolio!

Btw I think this might be a browser error, I'll try to debug this later, but anyways for some reason this is happening in my browser (brave):
image

you might want to check this out

**Pull request closed without merging**